src/app/zap-templates/zcl/data-model/chip/refrigerator-and-temperature-controlled-cabinet-cluster.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="SemanticTags" type="ENUM16">
+    <cluster code="0x0052"/>
+    <item value="0x4000" name="RapidCool"/>
+    <item value="0x4001" name="RapidFreeze"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Refrigerator And Temperature Controlled Cabinet</name>
+    <code>0x0052</code>
+    <define>REFRIGERATOR_AND_TEMPERATURE_CONTROLLED_CABINET_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster is an alias of the Mode Select cluster, defining additional semantics and
+      namespaced enumerated values for refrigerator and temperature controlled cabinet devices.</description>
+    <!-- Base data types -->
```

Comment:
Needs ClusterRevision bits.

Reply:
Updated in [0d6b3d89be2beba6d443811cafdce3d8e88a11c1](https://github.com/project-chip/connectedhomeip/pull/26508/commits/0d6b3d89be2beba6d443811cafdce3d8e88a11c1)